index.html

```diff
@@ -5099,7 +5099,7 @@ <h2><span class="chunk">acTL</span>
 
   </section>
 
-  <section>
+  <section id="11fcTL">
```

Comment:
Since this is a new section, if we want to use a hard-coded fragment instead of the ReSpec-generated one (from the `<h2>`) can we get rid of the "11"?

Reply:
... also we cannot be sure that respec will always use the same algorithm to generate section IDs, so I think we still need to include an `id` attribute on sections.

Reply:
But the "fcTL" chunk wasn't in previous editions of the spec. There are no links using that anchor.

Agreed on not relying on the generated section IDs being guaranteed consistent. I don't expect it to change, but I also never read that promise anywhere.

Reply:
> But the "fcTL" chunk wasn't in previous editions of the spec. There are no links using that anchor.

What about the following?

https://github.com/w3c/PNG-spec/blob/0184eda474506a9bf08128e4e7763f9ecdec6929/index.html#L2248

Reply:
The fcTL chunk is part of APNG, which is new as of 3rd Edition. The 2nd Edition doesn't have it.

Someone *could* link directly into the work-in-progress GitHub spec. But that isn't promised to be stable. It would be a bad decision for them to rely on that. I think we should have free reign to break things while they're work-in-progress.

Reply:
Oh wait. I think I misunderstood you.
I think you were showing there is already work-in-progress links to "#11fcTL".
I would like to change those existing links.

(If we want to do it in a separate pull request, that is fine.)

Reply:
> I think you were showing there is already work-in-progress links to "#11fcTL".

Yes, not by me though :)

Happy to change it to something more expressive. What do you suggest?

Reply:
"fcTL-chunk"?

Reply:
Fixed at https://github.com/w3c/PNG-spec/pull/177/commits/073c20a1c812fe781cf3af258b1633bd1b4557f3